django_mojeid/models.py

```diff
@@ -47,7 +48,7 @@ def __init__(self, *args, **kwargs):
 
         # Generate default salt
         if not 'salt' in kwargs:
-            kwargs['salt'] = os.urandom(30).encode('base64')[:-1]
+            kwargs['salt'] = base64.b64encode(os.urandom(30))
```

Comment:
https://usertech.atlassian.net/browse/KNJ-3015

File "/srv/apps/KNJ/app/var/ve/lib/python3.5/site-packages/django_mojeid/models.py" in __init__
  49.             kwargs['salt'] = os.urandom(30).encode('base64')[:-1]

Exception Type: AttributeError at /mojeid/callback/
Exception Value: 'bytes' object has no attribute 'encode'